### DISCUSSION TITLE
Simon, please add a PlantUML renderer. Fans, upvote!

### DISCUSSION THREAD

**Discussion by OlegKunitsyn** [structurizr/lite#128] [Ideas]
### Description

PlantUML can enhance documentation significantly and support non-C4 diagrams.

```plantuml
@startuml
!include workflow.puml
@enduml
```

### Priority

I have no budget and there's no rush, please add this feature for free

**Comment by simonbrowndotje** [comment]
Your request is a little vague, but rendering PlantUML is already possible:

- Image views -> https://docs.structurizr.com/ui/diagrams/image-view
- Documentation (via a plugin) -> https://github.com/structurizr/examples/tree/main/dsl/plantuml-and-mermaid
